### PR TITLE
load required filter modules for EC in pulseaudio

### PR DESCRIPTION
**Pull request by ad31c0**

### CODE REVIEW

File: service/platform/index.js [hunk 1]
```diff
@@ -205,6 +221,8 @@ class ServerPlatform extends Tp.BasePlatform {
             this._pulse = new PulseAudio();
             this._pulse.on('error', (err) => { console.error('error on PulseAudio', err); });
 
+            this._ensurePulseConfig();
```
Comment (gcampax) [inline service/platform/index.js:224]: Should we have a .catch here? In case loading modules fail, we don't want to crash.

Comment (ad31c0) [reply service/platform/index.js]: yeah, added just to be sure we don't crash with modules listing. Loading failure doesn't throw an exception.

**Review by gcampax — APPROVED** [review]
Looks good

**Comment by ad31c0** [comment]
Fixes #160

**Pull request merged**